### PR TITLE
en: add description of task config `ignore-checking-items`

### PR DESCRIPTION
**Pull request by ireneontheway**
### What is changed, added or deleted? (Required)

add a description of task config `ignore-checking-items`, part of https://github.com/pingcap/docs-dm/pull/200

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/177
- Other reference link(s): https://github.com/pingcap/docs-dm/pull/200

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

File: en/precheck.md [hunk 1]
```diff
@@ -64,3 +64,20 @@ Upstream and downstream database users must have the corresponding read and writ
             - The auto increment primary key exists in the sharded tables and its column type *is* bigint, but column mapping *is not* configured.
 
         - The check succeeds in other conditions except the two above.
+
+### Disable checking items
+
+DM checks items according to the task type, and users can use `ignore-checking-items` in the task configuration file to disable checking items. `ignore-checking-items` is a list, where values include:
```
Comment (lance6716) [inline en/precheck.md:70]: > where values include

"element" is often used to describe "unit forms a list", so I may advise "whose elements could be" or something

Comment (ireneontheway) [reply en/precheck.md]: I've updates in https://github.com/pingcap/docs-dm/pull/293/commits/549b72176bcc1df18692ea6cb2e8b2cf9e83c30a

**Comment by lance6716** [comment]
LGTM

File: en/task-configuration-file-full.md [hunk 2]
```diff
@@ -31,6 +35,7 @@ remove-meta: false              # Whether to remove the `meta` information (`che
 timezone: "Asia/Shanghai"       # The timezone.
 case-sensitive: false           # Determines whether the schema/table is case-sensitive.
 online-ddl-scheme: "gh-ost"     # Only "gh-ost" and "pt" are currently supported.
+ignore-checking-items: []       # Do not disable any checking items.
```
Comment (TomShawn) [inline en/task-configuration-file-full.md:38]:
```suggestion
ignore-checking-items: []       # No element, which means not to disable any checking items.
```

File: en/precheck.md [hunk 3]
```diff
@@ -64,3 +64,20 @@ Upstream and downstream database users must have the corresponding read and writ
...
         - The check succeeds in other conditions except the two above.
+
+### Disable checking items
+
+DM checks items according to the task type, and users can use `ignore-checking-items` in the task configuration file to disable checking items. `ignore-checking-items` is a list whose elements can be:
+
+| Element  | Description   |
+| :----  | :-----|
+| all | Disable all the checks |
+| dump_privilege | Disable the check of dump-related privileges of the upstream MySQL instance user |
```
[the remark this answers is not in the document]
Comment (ireneontheway) [reply en/precheck.md:75]: Is it ok to use "Disable checking" to keep consistency with other items?

[the remark this answers is not in the document]
Comment (TomShawn) [reply en/precheck.md:75]: Yes. If the clause following `disable the check` is longer, `disable checking` is better. BTW, `check for sth` might be more accurate.

File: en/precheck.md [hunk 4]
```diff
@@ -64,3 +64,20 @@ Upstream and downstream database users must have the corresponding read and writ
...
+### Disable checking items
+
+DM checks items according to the task type, and you can use `ignore-checking-items` in the task configuration file to disable checking items. The list of element options for `ignore-checking-items` is as follows:
+
+| Element  | Description   |
+| :----  | :-----|
+| all | Disable all checks |
+| dump_privilege | Disable checking dump-related privileges of the upstream MySQL instance user |
+| replication_privilege | Disable checking replication-related privileges of the upstream MySQL instance user |
+| version | Disable the check of upstream database version |
```
Comment (TomShawn) [inline en/precheck.md:77]:
```suggestion
| version | Disable checking the upstream database version |
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**